src/views/ev3/ev3.scss

```diff
@@ -5,6 +5,10 @@
         background-color: $ui-orange;
         background-image: url("/images/ev3/ev3-pattern.svg");
     }
+    .headline-icon {
```

Comment:
this keeps the ev3 icon slightly smaller than the others, making the size explicitly match the design